roles/horizon/meta/main.yml

```diff
@@ -5,11 +5,8 @@ dependencies:
     project_name: horizon
     git_mirror: "https://github.com/blueboxgroup"
     project_rev: "{{ horizon.source.rev }}"
-    virtualenv: "{{ horizon.virtualenv }}"
-    python_requirements:
-      - { name: functools32 }
-      - { name: python-memcached, version: '1.48' }
-      - { name: stevedore, version: '1.5.0' }
```

Comment:
virtualenv isn't needed anymore...    I messed up the requirements ( which are now listed in defaults )  but have fixed them to match this now.